addons/settings/BW_Settings.inc.sqf

```diff
@@ -108,11 +108,14 @@ if (["diwako_dui_main"] call ACEFUNC(common,isModLoaded)) then { _settings appen
 ["diwako_dui_indicators_range", 20], // User-Setting
 
 // DUI - Squad Radar - Radar
-["diwako_dui_compassRange", 35, true], // User-Setting
+["diwako_dui_compassRange", 50, true], // User-Setting (this one sets the starting range)
```

Comment:
maybe we should not set and force this and leave it a user setting?
the `limit` setting is doing what we wanted

Reply:
fix'd!